src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -126,6 +137,18 @@
                 data-cy="terraLaunchWith"
                 ><mat-icon svgIcon="terra"></mat-icon> Terra &raquo;</a
               >
+              <a
+                mat-raised-button
+                class="old-launch-with-button"
+                [matTooltip]="elwaziToolTip$ | async"
+                target="_blank"
+                rel="noopener"
+                [disabled]="disableElwaziPlatform$ | async"
```

Comment:
Because elWazi is a Terra platform, you could just use `disableTerraPlatform$` like the AnVIL and BDCat buttons do.